ui/job-view/details/BugFiler.jsx

```diff
@@ -264,25 +264,27 @@ export class BugFilerClass extends React.Component {
       if (lowerJobGroupName.includes('web platform')) {
         failurePath = failurePath.startsWith('mozilla/tests')
           ? `testing/web-platform/${failurePath}`
-          : `testing/web-platform/tests/${failurePath}`;
+          : `testing/web-platform/tests${failurePath}`;
```

Comment:
At least for the web platform jobs I've encountered today, they all started with a '/' so this used to double up as "//".